FastSimulation/Calorimetry/python/Calorimetry_cff.py

```diff
@@ -262,22 +263,13 @@
             timeShiftHF = cms.vdouble(50.7, 52.5, 52.9, 53.9, 54.5, 55.1, 55.1, 55.7, 55.9, 56.1, 56.1, 56.1, 56.5),
             ),
         HFShower           = cms.PSet(
-            ProbMax          = cms.double(1.0),
-            CFibre           = cms.double(0.5),
-            OnlyLong          = cms.bool(True)
+            HFShowerBlock     = HFShowerBlock
```

Comment:
This leads to "long distance interaction" that is very confusing in most cases. If anyone does e.g. `g4SimHits.HFShower.HFShowerBlock.ProbMax = 0.1`, that change will be visible here as well (because the `HFShowerBlock` PSet object is the same).

We typically advise against that behavior and do
```suggestion
            HFShowerBlock     = HFShowerBlock.clone()
```
instead.

If you want to be able to set value in `HFShowerBlock` PSet once and have the change to propagate to both here and `g4SimHits`, best would be to do
```suggestion
            HFShowerBlock     =  cms.PSet(refToPSet_ = cms.string("HFShowerBlock"))
```
and make sure the top-level `HFShowerBlock` gets attached to `cms.Process` (as it current does). Then the intention is clear, and the connection is visible e.g. `edmConfigDump`. The C++ side doesn't need any change.

Reply:
@bsunanda , let us do like this